x-pack/platform/plugins/private/discover_enhanced/test/scout/ui/fixtures/index.ts

```diff
@@ -21,24 +21,10 @@ export interface ExtScoutTestFixtures extends ScoutTestFixtures {
   };
 }
 
-export const test = baseTest.extend<ExtScoutTestFixtures, ScoutWorkerFixtures>({
-  pageObjects: async (
-    {
-      pageObjects,
-      page,
-    }: {
-      pageObjects: ExtScoutTestFixtures['pageObjects'];
-      page: ExtScoutTestFixtures['page'];
-    },
-    use: (pageObjects: ExtScoutTestFixtures['pageObjects']) => Promise<void>
-  ) => {
-    const extendedPageObjects = {
-      ...pageObjects,
-      demo: createLazyPageObject(DemoPage, page),
-    };
-
-    await use(extendedPageObjects);
-  },
+export const test = createTest<{
```

Comment:
This is the result I desired, a simple and terse way to extend the set of page objects. The createTest function is dense and difficult to read but the code it replaces didn't have any more meaning as best I could tell.